### PR TITLE
Copying indexing change into IM4K2 PPM

### PR DESCRIPTION
**Pull request by KaushikMalapati**
## Description

Putting same indexing changes from https://github.com/pcdshub/lcls-twincat-common-components/pull/74 into the IM4K2 custom PPM function blocks

## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive comments
- [ ] Test suite passes locally
- [ ] Libraries are set to fixed versions and not ``Always Newest``
- [ ] Code committed with ``pre-commit`` (alternatively ``pre-commit run --all-files``)

### CODE REVIEW

File: plc-kfe-rix-motion/kfe_rix_motion/kfe_rix_motion.plcproj [hunk 1]
```diff
@@ -191,7 +191,7 @@
       <Resolution>LCLS General, 2.10.0 (SLAC)</Resolution>
     </PlaceholderResolution>
     <PlaceholderResolution Include="lcls-twincat-common-components">
-      <Resolution>lcls-twincat-common-components, 3.8.0 (SLAC)</Resolution>
+      <Resolution>lcls-twincat-common-components, 0.0.0 (SLAC)</Resolution>
```
Comment (ZLLentz) [inline plc-kfe-rix-motion/kfe_rix_motion/kfe_rix_motion.plcproj:194]: It looks good, but maybe I should tag common components again and you can bump this number up to 3.8.1

Comment (KaushikMalapati) [reply plc-kfe-rix-motion/kfe_rix_motion/kfe_rix_motion.plcproj:194]: Sure, I just merged common components

Comment (ZLLentz) [reply plc-kfe-rix-motion/kfe_rix_motion/kfe_rix_motion.plcproj:194]: 3.8.1 with your fix is now on the prog node

**Comment by KaushikMalapati** [comment]
Did someone turn off plc-kfe-rix-motion? I tried to activate configuration with only a version change to 3.8.1 for common-components but couldn't and then saw the plc was unpingable. I was able to login an hour ago and the plc seemed fine.

**Comment by jyotiphy** [comment]
why would someone turn off the PLC ? It's not pinging. Going to power-cylce it.

**Comment by jyotiphy** [comment]
Remote power-cycle isn't helping, which is very strange!  :(  Can you go to FEE rack room and power-cycle it manually and see the status LEDs ?  @KaushikMalapati

**Comment by KaushikMalapati** [comment]
@jyotiphy @ZLLentz, can you think of what I did that would have caused the plc to go offline like this? It's currently running now with the same code that I tried earlier and the only thing I changed was pinning a new version of common components? (I initially had https://github.com/pcdshub/lcls-twincat-common-components/pull/74 as 0.0.0 but the code is the same) After activating configuration, I tried logging in but could not and then saw it was unpingable.

**Comment by ZLLentz** [comment]
I've had scattered rumors from various team members that the BSD PLCs go offline after the Nth activate config. I've not seen this myself nor been able to verify it.

The desired resolution for these sorts of problems will be to do a coordinated upgrade of the XAE and runtimes.

**Comment by jyotiphy** [comment]
I didn't know about this rumour 🤔  PLC came back alive after manual power-cycle.

**Comment by jyotiphy** [comment]
@ZLLentz can you please review, approve and merge this PR ?

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
Let me know when it's time for me to help look at the associated delta-updates-not-working issue

**Comment by jyotiphy** [comment]
@ZLLentz I tested this again after deploying master branch and now it seems to be working! Not sure what was the issue yesterday but all seems good now. Its moving as expected for additional 7mm (both directions).

**Comment by ZLLentz** [comment]
Ok great! Good job everyone!

**Pull request merged**